event/source/cpp/src/KaaDemo.cpp

```diff
@@ -172,35 +172,33 @@ class ChatMenu {
 
 private:
     bool processCommand(const std::string &command) {
```

Comment:
I propose to put all "if" to one construction.
Example:
bool processCommand(const std::string &command) {
        if (command.compare("1") == 0) {
            return command_join();
        } else if (command.compare("2") == 0) {
            return command_create();
        }  else if (command.compare("3") == 0) {
            return command_delete();
        } if (command.compare("4") == 0) {
            printRooms();
            return true;
        } else if (command.compare("5") == 0) {
            return false;
        } else {
            std::cout << "Unknown command " << command << '\n';
        }
        return true;
    }